dist.ini

```diff
@@ -1,5 +1,5 @@
 name = Rex
-version = 1.4.0
+version = 1.4.0_01
```

Comment:
I think setting the `release_status` below to `testing` will fix the test failure we got.